sources/packages/web/src/components/aest/institution/modals/ApproveProgramModal.vue

```diff
@@ -1,71 +1,105 @@
 <template>
-  <modal-dialog-base
-    title="Approve program"
-    :showDialog="showDialog"
-    @dialogClosed="dialogClosed"
-  >
-    <template v-slot:content>
-      <formio
-        formName="approveEducationProgram"
-        @submitted="submitForm"
-        @loaded="formLoaded"
-      ></formio>
-    </template>
-    <template v-slot:footer>
-      <check-permission-role :role="Role.InstitutionApproveDeclineProgram">
-        <template #="{ notAllowed }">
-          <footer-buttons
-            primaryLabel="Approve now"
-            @primaryClick="approveProgram"
-            @secondaryClick="dialogClosed"
-            :disablePrimaryButton="notAllowed"
-          />
-        </template>
-      </check-permission-role>
-    </template>
-  </modal-dialog-base>
+  <v-form ref="approveProgramForm">
+    <modal-dialog-base
+      title="Approve program"
+      :showDialog="showDialog"
+      max-width="730"
+    >
+      <template #content>
+        <error-summary :errors="approveProgramForm.errors" />
+        <div class="pb-2">
+          <span class="label-value"
+            >Outline the reasoning for approving this program. This will be
+            stored in the institution profile notes.</span
+          >
+        </div>
+        <!-- TODO Date picker is not available in the vuetify 3 version, so temporary usage of textfield and regex-->
+        <span class="label-bold mt-2">Effective end date</span>
+        <v-text-field
+          class="mt-2"
+          label="yyyy-MM-dd"
+          v-model="formModel.effectiveEndDate"
+          variant="outlined"
+          :rules="[
+            (v) =>
+              v.match(/^\d{4}-\d{2}-\d{2}$/) ||
+              'Effective end date is not in right format',
+          ]"
+        />
+        <div class="pb-2">
+          <span class="label-bold">Notes</span>
+        </div>
+        <v-textarea
+          label="Long text..."
```

Comment:
this should be actual label. there will be different property for placeholders. pls use that